### DISCUSSION TITLE
Konkrete Version auslesen

### DISCUSSION THREAD

**Discussion by DaReCo12** [stephanstapel/ZUGFeRD-csharp#417] [General]
Hallo Stephan,

ich nutze die lib um ZUGFeRD XML auszulesen. Gibt es hier eine einfache Möglichkeit, sich die genaue Versionsnummer der Datei ausgeben zulassen? Sprich ob es sich um Version 1.0, 2.0, 2.1,... Handelt?

Freundliche Grüße

**Comment by stephanstapel** [comment]
Hallo @DaReCo12 ,
nein, leider nicht. Es gibt lediglich:
https://github.com/stephanstapel/ZUGFeRD-csharp/blob/846db0bb15a8001bf391d76d45682e670606b26f/ZUGFeRD/InvoiceDescriptor.cs#L378

Ist aber eine interessante Frage: was ist Dein Anwendungsfall dafür?

**Reply by DaReCo12** [reply]
Morgen,


danke für die schnelle Antwort. Wenn die ZUGFeRDVersion.Version20 = 2.0 entspricht kann ich diese auch verwenden und umwandeln.
Ich Benötige Version die Weiterverarbeitung der Datei. Die ZUGFeRD XML wird nach meinem Programmdurchlauf noch gemappt. Die Mappings werden durch diese Versionsnummer bestimmt.

Grüße
